Makefile.cflags

```diff
@@ -51,6 +51,9 @@ endif
 # Forbid common symbols to prevent accidental aliasing.
 CFLAGS += -fno-common
 
+# Enable all default warnings
+CFLAGS += -Wall
```

Comment:
Actually I am not really sure if this is the right place to add this argument. It prevents the user from writing crappy code, which they might not take kindly.